i18n/locales/en.json

```diff
@@ -52,6 +52,10 @@
   "components.footer.links.github": "GitHub",
   "components.prevnextlink.next": "NEXT",
   "components.prevnextlink.previous": "PREV",
+  "docs.api.addedIn": "Added in: {version}",
```

Comment:
The path of translation keys should be: `components.api.apiChanges.{keyName}`

Reply:
ok